src/Logger/Handler/RecordChangeHandler.php

```diff
@@ -211,4 +210,40 @@ private function initialize()
         $this->allowed = ['INSERT', 'UPDATE', 'DELETE'];
         $this->initialized = true;
     }
+
+    /**
+     * @param array $a
+     * @param array $b
+     *
+     * @return array [key, left, right][]
+     */
+    private function diff(array $a, array $b)
```

Comment:
Erm, you might have forgotten about B.C on [`Bolt\DeepDiff`](https://github.com/bolt/bolt/blob/release/3.3/src/DeepDiff.php#L25)